network/src/peer/mod.rs

```diff
@@ -321,7 +321,19 @@ where
         trace!("Received message from Peer {}", self.peer_id().short_str());
         // Read inbound message from stream.
         let message = message.freeze();
-        let message: NetworkMessage = lcs::from_bytes(&message)?;
+        let message = match lcs::from_bytes(&message) {
+            Ok(message) => message,
+            Err(err) => {
+                // Don't bother returning errors for tiny messages
+                if message.len() >= 2 {
```

Comment:
is this 2 bytes?

Reply:
Yeah, if you notice, this is some of my lowest quality code... I only want to return meaningful error messages :P

Reply:
What is the specification for returning errors?  Optional but recommended?  Only if the message is bigger than 2 bytes?  I took a quick look at the network specs but couldn't find it.

Reply:
Great question, considering the amount of effort it would take to make it mandatory, I think we can say say it is optional assuming that it is parseable. For non-parseable errors, we only return errors for those of 2 bytes or more otherwise they aren't well-formed enough to provide any meaningful response.